From 4ae244cab4fe715fc2729e906b7dda3075fbbac3 Mon Sep 17 00:00:00 2001
From: Philip Cuadra <philipcuadra@google.com>
Date: Fri, 3 Aug 2018 11:22:11 -0700
Subject: [PATCH 27/80] Set rlimit rtprio for cameraserver

Set rlimit rtprio for cameraserver to allow it to inherit rt priority
via binder priority inheritance.

Bug:  80502612
Test: confirmed rt prio inheritance via systrace
Change-Id: I916370981690126c7c5302699c7c1bf5b6052685
---
 camera/cameraserver/cameraserver.rc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/camera/cameraserver/cameraserver.rc b/camera/cameraserver/cameraserver.rc
index fea5a1d5c..a9aae0b15 100644
--- a/camera/cameraserver/cameraserver.rc
+++ b/camera/cameraserver/cameraserver.rc
@@ -4,3 +4,4 @@ service cameraserver /system/bin/cameraserver
     group audio camera input drmrpc
     ioprio rt 4
     writepid /dev/cpuset/camera-daemon/tasks /dev/stune/top-app/tasks
+    rlimit rtprio 10 10
-- 
2.17.1

